.github/workflows/doc.yml

```diff
@@ -0,0 +1,26 @@
+name: Build LaTeX documentation
+on: [push]
```

Comment:
We probably want to restrict to pushes to `main`?  Otherwise also pushes to `gh-doc` (and any other branch) would trigger this, causing an interesting loop.

Reply:
I did mean to do this – left as `on [push]` so I could test the [action](https://github.com/t-young31/ifdtd/actions/runs/2516126147), but forgot to change it back. thanks 👍🏼